optimum/intel/ipex/modeling_base.py

```diff
@@ -160,9 +160,17 @@ def __init__(
             self._init_warmup()
 
     @classmethod
-    def _from_transformers(
+    def _from_transformers(cls, *args, **kwargs):
+        logger.warning(
+            "_from_transformers will be removed in the future, please remove the export arguments in the API."
+        )
```

Comment:
We should keep `export` to know when the model needs to be converted to torchscript no ? Especially if we want to have default compatibility across devices (xpu / cpu)